pages/common/git-flow.md

```diff
@@ -0,0 +1,24 @@
+# git flow
+
+> A collection of Git extensions to provide high-level repository operations for Vincent Driessen's branching model.
+> More information: <https://github.com/nvie/gitflow>.
+
+- Start using git-flow by initializing it inside an existing git repository:
+
+`git flow init`
+
+- Start developing a new feature. It creates a new branch based on develop:
```

Comment:
How about "Start developing a new feature by creating a new branch" ?

Reply:
I think "Start developing on a feature branch based on `develop`" is good